versions/draft-3/SPEC.md

```diff
@@ -891,14 +891,16 @@ output {
 ```
 
 
-Globs can be used to define outputs which contain many files.  The glob function generates an array of File outputs:
+Globs can be used to define outputs which might contain zero, one, or many files. The glob function therefore returns an array of File outputs:
 
 ```
 output {
   Array[File] output_bams = glob("*.bam")
 }
 ```
 
+The array of `File`s returned matches the bash evaluation of the glob from the execution directory the context of the bash version installed in the docker image running the task. In other words, it contains the files which would be matched by running `ls <glob>` from the task's execution directory.
```

Comment:
Is something off in the first sentence or is it just me ?

Reply:
Hmm, you're right, it is a little bit strange on second reading. I'll try to break it into a couple of sentences instead